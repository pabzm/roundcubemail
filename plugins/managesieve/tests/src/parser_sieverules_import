## Generated by Roundcube Webmail SieveRules Plugin ##
require ["imap4flags","fileinto"];

# rule:[Out of Office]
# disabledRule:[a:5:{s:4:"join";b:0;s:4:"name";s:13:"Out of Office";s:8:"disabled";s:1:"1";s:5:"tests";a:1:{i:0;a:1:{s:4:"type";s:4:"true";}}s:7:"actions";a:1:{i:0;a:10:{s:4:"type";s:8:"vacation";s:4:"days";s:1:"3";s:7:"subject";s:13:"Out of office";s:11:"origsubject";s:1:"1";s:4:"from";s:0:"";s:9:"addresses";s:0:"";s:6:"handle";s:0:"";s:3:"msg";s:16:"Went for holiday";s:7:"htmlmsg";b:0;s:7:"charset";s:5:"UTF-8";}}}]
# rule:[Spam]
# disabledRule:[a:5:{s:4:"join";b:0;s:4:"name";s:4:"Spam";s:8:"disabled";s:1:"1";s:5:"tests";a:1:{i:0;a:5:{s:3:"not";s:0:"";s:4:"type";s:6:"header";s:8:"operator";s:8:"contains";s:6:"header";s:7:"Subject";s:6:"target";s:12:"****SPAM****";}}s:7:"actions";a:1:{i:0;a:3:{s:4:"type";s:8:"fileinto";s:6:"create";b:0;s:6:"target";s:4:"Junk";}}}]
# rule:[Stop]
# disabledRule:[a:5:{s:4:"join";b:0;s:4:"name";s:4:"Stop";s:8:"disabled";s:1:"1";s:5:"tests";a:1:{i:0;a:5:{s:3:"not";s:0:"";s:4:"type";s:6:"header";s:8:"operator";s:8:"contains";s:6:"header";s:7:"Subject";s:6:"target";s:18:"contains this text";}}s:7:"actions";a:1:{i:0;a:1:{s:4:"type";s:4:"stop";}}}]
