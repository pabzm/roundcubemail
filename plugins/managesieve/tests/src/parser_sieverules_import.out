## Generated by Roundcube Webmail SieveRules Plugin ##

require ["fileinto","vacation"];
# rule:[Out of Office]
if false # true
{
	vacation :days 3 :subject "Out of office" "Went for holiday";
}
# rule:[Spam]
if false # header :contains "Subject" "****SPAM****"
{
	fileinto "Junk";
}
# rule:[Stop]
if false # header :contains "Subject" "contains this text"
{
	stop;
}
